plugin-script/src/main/java/io/kestra/plugin/scripts/exec/AbstractExecScript.java

```diff
@@ -33,15 +32,14 @@ public abstract class AbstractExecScript extends Task implements RunnableTask<Sc
         description = "Only used if the `taskRunner` property is not set"
     )
     @PluginProperty
-    @NotEmpty
+    @NotNull
     protected RunnerType runner = RunnerType.DOCKER;
 
     @Schema(
         title = "The task runner to use.",
         description = "Task runners are provided by plugins, each have their own properties."
     )
     @PluginProperty
-    @Valid
```

Comment:
AFAIK it should be kept or validation will not be done inside the TaskRunner attribute itself